datatypes/src/primitives/time_interval.rs

```diff
@@ -213,9 +213,19 @@ impl TimeInterval {
     ///     })
     /// );
     /// ```
+    #[allow(unstable_name_collisions)] // TODO: remove when `clamp` is stable
     pub fn to_geo_json_event(&self) -> serde_json::Value {
-        let start_date: DateTime<Utc> = Utc.timestamp_millis(self.start());
-        let end_date: DateTime<Utc> = Utc.timestamp_millis(self.end());
+        let min_visualizable_value = -8_334_632_851_200_001 + 1; // -262144-01-01T00:00:00+00:00
+        let max_visualizable_value = 8_210_298_412_800_000 - 1; // +262143-12-31T23:59:59.999+00:00
```

Comment:
use string constant instead of parsing the same date over and over again

Reply:
e0e9ca2